src/brasil/gov/tiles/tiles/navigation.py

```diff
@@ -0,0 +1,70 @@
+# -*- coding: utf-8 -*-
+from Acquisition import aq_parent
+from brasil.gov.tiles import _
+from collective.cover.tiles.base import IPersistentCoverTile
+from collective.cover.tiles.base import PersistentCoverTile
+from plone import api
+from plone.dexterity.interfaces import IDexterityContent
+from Products.Five.browser.pagetemplatefile import ViewPageTemplateFile
+from zope.interface import implementer
+
+
+class INavigationTile(IPersistentCoverTile):
+    """Display a navigation of items."""
+
+
+@implementer(INavigationTile)
+class NavigationTile(PersistentCoverTile):
+    """Display a navigation of items."""
+
+    index = ViewPageTemplateFile('templates/navigation.pt')
+    short_name = _(u'msg_short_name_navigation', default=u'Navigation')
+
+    is_configurable = False
+    is_droppable = False
+    is_editable = False
+
+    def __init__(self, context, request):
+        super(NavigationTile, self).__init__(context, request)
+        self._setup()
```

Comment:
this kind of code needs to be outside of the `__init__` method or it will raise `NotFound` on errors; use the `__call__` method instead.

Reply:
method __call__ not function in tile.

Reply:
I'll take a look on it later.

Reply:
@hvelarde 
Não estava funcionando no início, vi que o tile estava com cache.
Já foi implementado com o método __call__.